### PR TITLE
Get rid of jsx syntax

### PR DESCRIPTION
**Pull request by NicolasParada**
Solves #91

### CODE REVIEW

**Comment by developit** [comment]
Thanks!

File: src/index.js [hunk 1]
```diff
@@ -236,9 +236,9 @@ class Router extends Component {
 }
 
 
-const Route = ({ component:RoutedComponent, url, matches }) => (
-	<RoutedComponent {...{url, matches}} />
-);
+const Route = ({ component: RoutedComponent, url, matches }) => {
+	return h(RoutedComponent, { url, matches });
```
Comment (developit) [inline src/index.js]: Can just call this `component` now. (I don't mind changing after merge if you're good)

Comment (NicolasParada) [inline src/index.js]: Ok

**Comment by developit** [comment]
its beautiful

**Pull request merged**